DEV_GUIDE.md

```diff
@@ -0,0 +1,102 @@
+# Development Guide for Kroxylicious Junit5 Extension
+
+This document gives a detailed breakdown of the various build processes and options for building the Kroxylicious Junit5 Extension from source.
+
+## Build Prerequisites
+
+- [JDK](https://openjdk.org/projects/jdk/17/) (version 17 and above) - Maven CLI
+- [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
+- [`docker`](https://docs.docker.com/install/) or [`podman`](https://podman.io/docs/installation) - Docker or Podman
+
+> :warning: **If you are using Podman please see the notes below**
+> 
+## JDK target
+
+The project targets language Java level 17.
```

Comment:
Maybe better to say the project **production** sources target 17? As the tests depend on JDK 20 for SNI stuff.

Reply:
There's no SNI in the unit5 extension.  The whole project is Java 17.

Reply:
Oh right, I'm think of the wrong project!